src/main/resources/org/zaproxy/zap/extension/hud/resources/Messages.properties

```diff
@@ -3,14 +3,14 @@
 hud.api.action.log = Used by the HUD to log messages from the browser
 hud.api.action.recordRequest = Used by the HUD to cache a request the user wants to send in the browser
 hud.api.action.resetTutorialTasks = Reset the tutorial tasks so that they must be completed again
-hud.api.action.setOptionBaseDirectory = Set the base directory from which the HUD files are loaded 	
+hud.api.action.setOptionBaseDirectory = Set the base directory from which the HUD files are loaded
 hud.api.action.setOptionDevelopmentMode = Sets the boolean option 'Development mode'
-hud.api.action.setOptionEnabled = Sets whether the HUD is enabled or not
+hud.api.action.setOptionEnabled = Sets whether or not the HUD is enabled
 hud.api.action.setOptionInScopeOnly = Sets the boolean option 'In scope only'
 hud.api.action.setOptionRemoveCSP = Sets the boolean option 'Remove CSP'
-hud.api.action.setUiOption = Sets a UI option with the given key and value. The key must be 50 alphanumeric characters or less
+hud.api.action.setUiOption = Sets a UI option with the given key and value. The key must be 50 or fewer alphanumeric characters
 hud.api.view.getUiOption = Gets a UI option set using setUiOption. If the supplied key is not set then an empty string is returned
-hud.api.view.optionAllowUnsafeEval = Returns true if the 'Allow unsafe eval' option is set
+hud.api.view.optionAllowUnsafeEval = Returns true if the 'Allow unsafe-eval' option is set
```

Comment:
Should the UI label (line 45) be updated? Not exactly the same (using plural), but I guess the intent was that.